test/data/ansible/check_ostree.yaml

```diff
@@ -134,47 +134,12 @@
           set_fact:
             failed_counter: "{{ failed_counter | int + 1 }}"
 
-    # bios or uefi
-    - name: bios or uefi
-      stat:
-        path: /sys/firmware/efi
-      register: result_uefi
-
-    - set_fact:
-        device_name: /dev/vda2
-
-    - set_fact:
-        device_name: /dev/vda3
-      when: result_uefi.stat.exists
-
-    # Check if it is raw-image or simplified-installer, the disk are encrypted, and mount point starts with /dev/mapper/luks-
-    - name: check if it is simplified-installer or raw-image
+    - name: set mount point device name
       command: findmnt -r -o SOURCE -n /sysroot
       register: result_encrypted
```

Comment:
@yih-redhat Could you please rename the variable `result_encrypted`? This name is confusing in this case. Thanks.